augur/config.py

```diff
@@ -33,6 +33,34 @@
                     "model": "issues",
                     "repo_group_id": 0
                 },
+                {
+                    "all_focused": 1,
+                    "delay": 150000,
+                    "given": [
+                        "git_url"
+                    ],
+                    "model": "merge_requests",
+                    "repo_group_id": 0
+                },                        
+                {
+                    "all_focused": 1,
+                    "delay": 150000,
+                    "given": [
+                        "git_url"
+                    ],
+                    "model": "merge_request_commits",
+                    "repo_group_id": 0
+                },
+                        
+                {
+                    "all_focused": 1,
+                    "delay": 150000,
+                    "given": [
+                        "git_url"
+                    ],
+                    "model": "merge_request_files",
```

Comment:
`all_focused` and merge request terminology comments apply here too.

Reply:
This is also unresolved, for the same reasons as above.

Reply:
Same answer as above. Its been there over two years. Now is not the time to stop a pull request because of it. Ask @gabe-heim .